.gitignore

```diff
@@ -57,6 +57,9 @@ local_settings.py
 db.sqlite3
 project.db
 
+#Static generated files
+starlight/static/admin/
```

Comment:
You should add `starlight/static/bundles` to this as well, so we exclude the output that Webpack makes.